Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/AssetPage.tsx

```diff
@@ -563,7 +557,7 @@ export default function AssetPage(props: IProps) {
                 >
                     <div className="row">
                         <div className="col-8">
-                            <div className="row" style={{ maxHeight: innerHeight - 300, overflow:'auto' }}>
+                            <div className="row" style={{ maxHeight: innerHeight - 300, overflow: 'auto' }}>
```

Comment:
not  great idea to hardcode a height (`innerHeight - 300`) is there a reason that's needed? (e.g. in a Modal)

Reply:
I'm not sure, it seems very necessary, just places a scroll bar on that specific portion (excluding the pick list) of the modal instead of the entire modal